Assets/Scripts/Game/UserInterfaceWindows/DaggerfallUnitySaveGameWindow.cs

```diff
@@ -474,7 +487,8 @@ private void SaveLoadEventHandler(BaseScreenComponent sender, Vector2 position)
                     return;
                 }
 
-                LoadGame();
+                loadingLabel.Text = "Please wait...";
```

Comment:
This message should be added to _Text/DaggerfallUI.txt_ for support to localization. Then you can load it with `TextManager.GetText()`.

Reply:
Added a key to the text file.